### PR TITLE
RDoc-2358_shardedDbDocExt

### PR DESCRIPTION
**Pull request by reebhub**

### CODE REVIEW

File: Documentation/6.0/Raven.Documentation.Pages/sharding/document-extensions.markdown [hunk 1]
```diff
@@ -3,24 +3,119 @@
...
+  [Attachments](../document-extensions/attachments/what-are-attachments), 
+  [Time Series](../document-extensions/timeseries/overview), 
+  and [Revisions](../document-extensions/revisions/overview).  
+
+* From a user's point of view, document extensions behave similarly 
+  under sharded and non-sharded databases and are handled using the 
+  same API commands and Studio views.  
+
+* Document extensions are identified by the ID of their parent document, 
+  and are always stored in **the same bucket and shard** as the document.
```
Comment (karmeli87) [inline Documentation/6.0/Raven.Documentation.Pages/sharding/document-extensions.markdown:18]: same bucket.
bucket can move from shard to shard

File: Documentation/6.0/Raven.Documentation.Pages/sharding/document-extensions.markdown [hunk 2]
```diff
@@ -3,24 +3,119 @@
...
+## Precautions
+
+The main contribution of sharded databases is their ability to manage huge 
+volumes of data efficiently by serving it from multiple shards.  
+We should take extra care, then, to help the database maintain its ability 
+to divide the data between shards.  
+The following points relate to this issue.  
+
+* **Time Series**  
+  Some time series can get very large very quickly. As they reside in
```
Comment (karmeli87) [inline Documentation/6.0/Raven.Documentation.Pages/sharding/document-extensions.markdown:57]: I would drop the `very quickly` statement here

File: Documentation/6.0/Raven.Documentation.Pages/sharding/document-extensions.markdown [hunk 3]
```diff
@@ -3,24 +3,119 @@
...
+
+* **Time Series**  
+  Some time series can get very large very quickly. As they reside in 
+  a single bucket with their parent document, they cannot be spread between 
+  shards and may become hard to manage and use.  
+  We recommend keeping the number of time series added to each document 
+  fairly small, and using practices such as [rollup and retention](../document-extensions/timeseries/rollup-and-retention).  
+
+* **Revisions**  
+  Revisions in a large database may accumulate quickly, especially in
```
Comment (karmeli87) [inline Documentation/6.0/Raven.Documentation.Pages/sharding/document-extensions.markdown:64]: also here I wouldn't use `quickly`

File: Documentation/6.0/Raven.Documentation.Pages/sharding/document-extensions.markdown [hunk 4]
```diff
@@ -3,24 +3,119 @@
...
+We should take extra care, then, to help the database maintain its ability 
+to divide the data between shards.  
+The following points relate to this issue.  
+
+* **Time Series**  
+  Some time series can get very large very quickly. As they reside in 
+  a single bucket with their parent document, they cannot be spread between 
+  shards and may become hard to manage and use.  
+  We recommend keeping the number of time series added to each document 
+  fairly small, and using practices such as [rollup and retention](../document-extensions/timeseries/rollup-and-retention).
```
Comment (karmeli87) [inline Documentation/6.0/Raven.Documentation.Pages/sharding/document-extensions.markdown:61]: great point about the retention :)

**Review by karmeli87 — APPROVED** [review]
LGTM

File: Documentation/6.0/Raven.Documentation.Pages/sharding/document-extensions.markdown (hunk 1 above)
Comment (reebhub) [reply Documentation/6.0/Raven.Documentation.Pages/sharding/document-extensions.markdown:18]: done

File: Documentation/6.0/Raven.Documentation.Pages/sharding/document-extensions.markdown (hunk 2 above)
Comment (reebhub) [reply Documentation/6.0/Raven.Documentation.Pages/sharding/document-extensions.markdown:57]: done

**Pull request merged**